tests/test_sanity_import.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+
+import importlib
+
+te = importlib.util.find_spec('transformer_engine.pytorch')
```

Comment:
Why are we checking it this way instead of doing a normal `import transformer_engine` after completing build?

Reply:
We could do it that way and have custom error message via the `try`/`except` mechanism. There is no particular reason I guess :-)

Reply:
:+1:

Reply:
@ptrendx

Reply:
? I already changed that :-)